### DISCUSSION TITLE
Tablas en customtkinter

### DISCUSSION THREAD

**Discussion by feerxz** [TomSchimansky/CustomTkinter#1804] [Q&A]
Hola, soy nuevo aquí, estoy empezando a usar la librería customtkinter en python, estoy haciendo un proyecto para mi universidad, y la duda que tengo es cómo puedo crear una tabla usando esta librería, así como en tkinter es con ttk.Treeview, porque necesito mostrar datos de mis tablas de mi base de datos mysql. Espero me puedan ayudar

**Comment by avalon60** [comment]
You might want to check out:
[CTkTable](https://github.com/Akascape/CTkTable)
